src/utils/DynamicContent/generators/CurrentTime.ts

```diff
@@ -0,0 +1,23 @@
+import { TextGenerator } from '@src/utils/DynamicContent/generators/TextGenerator';
+import { Translator } from '@src/Translator';
+import { Ordinal } from '@src/utils/DynamicContent/formatters/Ordinal';
+import { Time } from '@src/utils/DynamicContent/formatters/Time';
+
+export class CurrentTime implements TextGenerator<Date> {
+	private readonly _translator: Translator;
+	private _ordinalFormatter: Ordinal;
+	private _timeFormatter: Time;
+
+	public constructor( translator: Translator, ordinalFormatter: Ordinal, timeFormatter: Time ) {
+		this._translator = translator;
+		this._ordinalFormatter = ordinalFormatter;
+		this._timeFormatter = timeFormatter;
+	}
+
+	public getText( date: Date ): string {
+		return this._translator.translate( 'date-month-time-' + ( date.getMonth() + 1 ), {
```

Comment:
The name of the class is a bit off from the implementation, which returns current date and time. I'm wondering if this isn't too much duplication and we'd be better (in terms of flexibility and DRY-ness) doing `{{currentDate}}, {{currenTime}}` in the banners instead of `{{ currentTime }}`. Although that would lead to an edge case at midnight if we don't update both date and time.

Reply:
I was thinking about that name alreaugh, lets do `currentDateAndTime`?